### ISSUE TITLE
Ops!

### ISSUE THREAD

**Issue by mixedbyte** [sindresorhus/caprine#2143]
Probably worth mentioning that this will not work if you do not have messenger on your phone. Wasted alot of time with this. 

---

Caprine 2.59.3
Electron 25.9.8
linux 6.7.5-1-default
Locale: en-US

**Comment by hsaladin** [comment]
I don't have messenger on my phone either (and never had since I don't have a smartphone) but until a few days ago Caprine used to work fine on my laptop computer running Ubuntu.
So, that is the reason why it is not mentioned that this will not work if you do not have messenger on your phone : it's because it's supposed to work !
It looks like the reason it has stopped working is because Meta has made some changes recently. Let's hope a solution will be found soon.

**Comment by aiman-al-masoud** [comment]
Indeed, you're not supposed to have the messenger app on your phone for Caprine to work, but I have recently experienced the same problem. It seems that if you:

- launch Caprine
- open dev tools (F12)
- set location.href to something other than the current page
- close and restart the app

Caprine will go back to displaying your chats, at least temporarily. I think the right kind of URL that Caprine should be opening on is something like: 'https://www.messenger.com/t/<CHAT_ID>'.

**Comment by Piscium** [comment]
For me messenger.com does not work on the browser, neither on desktop or phone, it asks for the phone app.
However "facebook.com/messages" works on the desktop (not phone).   Caprine is of course not a phone app.  I wonder if the url for caprine could be changed to "facebook.com/messages"? This would not work for people that have only a messenger account though. So perhaps a solution would be to make the url configurable by the user.

**Comment by alekksander** [comment]
> set location.href to something other than the current page

can You explain more ? my experience with dev tools is limited to css and performance
